vexcl/pointer_qualifiers.hpp

```diff
@@ -0,0 +1,121 @@
+#ifndef VEXCL_POINTER_QUALIFIERS_HPP
+#define VEXCL_POINTER_QUALIFIERS_HPP
+
+
+#include <vexcl/operations.hpp>
+/// \cond INTERNAL
+namespace vex {
```

Comment:
Please swap the above two lines. Its not a compilation error, but conceptually the namespace encloses the code tagged as `INTERNAL`.